pkg/k8s/apis/cilium.io/v1/types.go

```diff
@@ -74,6 +74,12 @@ type CiliumNetworkPolicyNodeStatus struct {
 
 	// LastUpdated contains the last time this status was updated
 	LastUpdated Timestamp `json:"lastUpdated,omitempty"`
+
+	// Revision contains the current policy revision of the node.
```

Comment:
Revision is policy revision of the repository which first implemented this policy

Reply:
Done